### PR TITLE
Revoke and grant admin endpoints

### PR DESCRIPTION
**Pull request by juliasaveliff**

### CODE REVIEW

File: rackcity/views/user_views.py [hunk 1]
```diff
@@ -213,6 +213,131 @@ def who_am_i(request):
...
 
+@api_view(['GET'])
+def i_am_admin(request):
+    """
+    Returns whether logged in user is an admin user.
+    """
+    if(request.user.is_staff):
+        return JsonResponse({"is_admin": True})
+    else:
+        return JsonResponse({"is_admin": False})
```
Comment (juliasaveliff) [inline rackcity/views/user_views.py]: moved this johnson from models to users where it belongs

Comment (benauriemma) [reply rackcity/views/user_views.py]: Did you just.......... `if true return true`

Comment (benauriemma) [reply rackcity/views/user_views.py]: Also I imagine it defaults to `status=HTTPStatus.OK` but idk if we want to say that explicitly?

Comment (benauriemma) [reply rackcity/views/user_views.py]: Did you just.......... `if true return true`

Comment (benauriemma) [reply rackcity/views/user_views.py]: Also I imagine it defaults to `status=HTTPStatus.OK` but idk if we want to say that explicitly?

Comment (juliasaveliff) [reply rackcity/views/user_views.py]: reverse uno card

**Pull request merged**